certif/app/components/add-student-list.js

```diff
@@ -22,6 +22,23 @@ export default class AddStudentList extends Component {
     return allCertifReportsAreCheck;
   }
 
+  get numberOfStudentsAlreadyCandidate() {
+    return this.args.certificationCandidates ? this.args.certificationCandidates.length : 0;
+  }
+
+  get showStickyBar() {
+    const thereIsAlreadySomeCandidates = this.numberOfStudentsAlreadyCandidate > 0;
+    return this.hasCheckedSomething || thereIsAlreadySomeCandidates;
+  }
+
+  get studentsSelectedInformation() {
+    const countStudents = (count, student) => student.isSelected ? count + 1 : count;
+    const numberOfStudentsSelected = this.args.studentList.reduce(countStudents, 0);
+    return numberOfStudentsSelected > 0
+      ? `${numberOfStudentsSelected} candidat(s) sélectionné(s)`
+      : 'Aucun candidat sélectionné';
+  }
```

Comment:
Je pense qu'un filter fait le job ici ?
```suggestion
  get studentsSelectedInformation() {
    const countStudents = (count, student) => student.isSelected ? count + 1 : count;
    const selectedStudents = this.args.studentList.filter((student) => student.isSelected);
    return selectedStudents.length > 0
      ? `${selectedStudents.length} candidat(s) sélectionné(s)`
      : 'Aucun candidat sélectionné';
  }
```